config/config.toml

```diff
@@ -24,6 +24,7 @@ inputFile = "depositsAndKeys.yaml"
 genesisTime = 298928929424
 ownedValidatorStartIndex = 0
 ownedValidatorCount = 4
+# startState = "initialState.ssz"
```

Comment:
we should remove the old file method where we read in keys and deposits and replace it with the initialState.ssz.


The config would look like:
```
[interop]
active = false
# "file" reads validator creds from file
# "mocked" generated validator cred deterministically
mode = "file"
inputFile = "initialState.ssz"
genesisTime = 298928929424
ownedValidatorStartIndex = 0
ownedValidatorCount = 4
```